proxy.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+const axios = require('axios').default;
+const cors = require('cors');
+require('dotenv').config();
+
+let getApiKey;
+
+function keyHolder(userSecret) {
+  let apiKey;
+
+  for (let i in userSecret) {
+    if (i === 'access_token') apiKey = userSecret[i];
+  }
+
+  function displayApiKey() {
+    return apiKey;
+  }
+  return displayApiKey;
+}
+
+const app = express();
+app.use(cors());
+
+app.get('/login', (req, res) => {
+  axios({
+    method: 'post',
+    url: `https://pfa.foreca.com/api/authorize/token`,
+    data: {
+      user: process.env.REACT_APP_AUTH_USERNAME,
+      password: process.env.REACT_APP_AUTH_PASSWORD
+    }
+  })
+    .then(response => {
+      console.log('API call succeeded!');
```

Comment:
please remove console logs used for debugging (console.error for logging errors is allowed)